README.md

```diff
@@ -41,6 +41,7 @@ Each country has a CO2 mass flow that depends on neighboring countries. In order
 Real-time electricity data is obtained using [parsers](https://github.com/corradio/electricitymap/tree/master/parsers)
 
 - Australia: [AREMI National Map](http://services.aremi.nationalmap.gov.au/aemo/v3/csv/all)
+- Western Australie: [AEMO] (http://wa.aemo.com.au/aemo/data/wa/infographic/facility-intervals-last96.csv)
```

Comment:
For consistency with the app, I believe we should write `Australia (Western)` or add it as a subitem.